test/facility-activation-transaction/post-facility-activation-transaction.api-test.ts

```diff
@@ -0,0 +1,283 @@
+import { ENUMS, PROPERTIES } from '@ukef/constants';
+import { AcbsGetFacilityResponseDto } from '@ukef/modules/acbs/dto/acbs-get-facility-response.dto';
+import { DateStringTransformations } from '@ukef/modules/date/date-string.transformations';
+import { withAcbsAuthenticationApiTests } from '@ukef-test/common-tests/acbs-authentication-api-tests';
+import { IncorrectAuthArg, withClientAuthenticationTests } from '@ukef-test/common-tests/client-authentication-api-tests';
+import { withFacilityIdentifierFieldValidationApiTests } from '@ukef-test/common-tests/request-field-validation-api-tests/facility-identifier-field-validation-api-tests';
+import { withNonNegativeNumberFieldValidationApiTests } from '@ukef-test/common-tests/request-field-validation-api-tests/non-negative-number-field-validation-api-tests';
+import { withStringFieldValidationApiTests } from '@ukef-test/common-tests/request-field-validation-api-tests/string-field-validation-api-tests';
+import { Api } from '@ukef-test/support/api';
+import { ENVIRONMENT_VARIABLES, TIME_EXCEEDING_ACBS_TIMEOUT } from '@ukef-test/support/environment-variables';
+import { CreateFacilityActivationTransactionGenerator } from '@ukef-test/support/generator/create-facility-activation-transaction-generator';
+import { GetFacilityGenerator } from '@ukef-test/support/generator/get-facility-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import nock from 'nock';
+
+describe('POST /facilities/{facilityIdentifier}/activation-transactions', () => {
+  const valueGenerator = new RandomValueGenerator();
+  const dateStringTransformations = new DateStringTransformations();
+  const facilityIdentifier = valueGenerator.facilityId();
+  const createFacilityActivationTransactionUrl = `/api/v1/facilities/${facilityIdentifier}/activation-transactions`;
+  const { servicingQueueIdentifier } = PROPERTIES.GLOBAL;
+
+  const { portfolioIdentifier } = PROPERTIES.GLOBAL;
+  const bundleIdentifier = valueGenerator.acbsBundleId();
+
+  const { facilitiesInAcbs } = new GetFacilityGenerator(valueGenerator, dateStringTransformations).generate({
+    numberToGenerate: 1,
+    facilityIdentifier,
+    portfolioIdentifier,
+  });
+  const effectiveDate = dateStringTransformations.removeTime(facilitiesInAcbs[0].OriginalEffectiveDate);
+  const borrowerPartyIdentifier = facilitiesInAcbs[0].BorrowerParty.PartyIdentifier;
+  const facilityInAcbs: AcbsGetFacilityResponseDto = facilitiesInAcbs[0];
```

Comment:
We can make a small change here - this when we are targeting a value from an array
```
const [facilityInAcbs]: AcbsGetFacilityResponseDto = facilitiesInAcbs;
```